runtime/browser/runtime_platform_util_linux.cc

```diff
@@ -43,7 +43,13 @@ void XDGUtil(const std::string& util, const std::string& arg) {
 }
 
 void XDGOpen(const std::string& path) {
-  XDGUtil("xdg-open", path);
+  GURL url(path);
+  if (url.is_valid()) {
+    std::string cmd = "xdg-open " + path;
+    std::system(cmd.c_str());
```

Comment:
The url like http://, https:// will use this, when open the external url in web browser, 'base::LaunchProcess' can not open the url in web browser when already has a running browser, so use 'std:system' instead.

I'll change to check whether the scheme is http or https.